go-controller/Makefile

```diff
@@ -26,7 +26,7 @@ OVN_SCHEMA_VERSION ?= v23.06.0
 ifeq ($(NOROOT),TRUE)
 C_ARGS = -e NOROOT=TRUE
 else
-C_ARGS = --cap-add=NET_ADMIN --cap-add=SYS_ADMIN
+C_ARGS = --cap-add=NET_ADMIN --cap-add=SYS_ADMIN --privileged
```

Comment:
I would have thought SYS_ADMIN would be enough :confused:

Reply:
Heh, to be honest I'm not 100% what triggered it... I think apparmor but did not verified :)